css/properties/background-attachment.json

```diff
@@ -77,11 +77,21 @@
               "opera_android": {
                 "version_added": "14"
               },
-              "safari": {
-                "version_added": false
-              },
+              "safari": [
+                {
+                  "version_added": "14",
+                  "partial_implementation": true,
+                  "notes": "<code>local</code> is recognized but has no effect due to [a bug](https://bugs.webkit.org/show_bug.cgi?id=219324)."
```

Comment:
This is failing CI. For reasons I only vaguely remember, we use specific formats of bug URLs. See the CI output for details: https://github.com/mdn/browser-compat-data/pull/10542/checks?check_run_id=2756698276#step:5:2338

Also, we don't use Markdown in the notes field. You'll have to use an old fashioned `<a href='…'>` link, I'm afraid.

Reply:
done ✅